workflow/automation/org/kisti/vm_gen.pbs

```diff
@@ -37,7 +37,7 @@ echo python $gmsim/Pre-processing/VM/vm_params2vm.py -t 18 $REL_NAME $VM_PARAMS_
 python $gmsim/Pre-processing/VM/vm_params2vm.py -t 18 $REL_NAME $VM_PARAMS_YAML -o $OUTPUT_DIR
 
 chmod g+rwXs -R $OUTPUT_DIR
-chgrp nesi00213 -R $OUTPUT_DIR
+chgrp `groups` -R $OUTPUT_DIR
```

Comment:
`groups` prints out all groups that a user is part of. `id -gn $username` only prints the primary group. (unsure if this will be a personal group or the group group though)

Reply:
This is a Nurion-specific change and a user is not a member of multiple groups.

Reply:
Experimentally they give the same result, however if one is guaranteed to return only one value, then we should use that one

Reply:
Sure. Fixed in the new commit, even though I know KISTI policy mandates only one group membership.